terraform/helm/aptos-node/values.yaml

```diff
@@ -124,7 +120,7 @@ fullnode:
   # -- Log level for the fullnode
   rust_log: info
   # -- Remote log level for the fullnode
-  rust_log_remote: debug,hyper=off
+  rust_log_remote: off
```

Comment:
this disables the remote logging functionality of the `aptos-logger` crate right? 
if so, might be useful to include some docs on how to configure that (setting the remote log level and the address)